src/bundle/Templating/Twig/UiConfigExtension.php

```diff
@@ -39,6 +42,25 @@ public function __construct(Environment $twig, Aggregator $aggregator)
      */
     public function getGlobals(): array
     {
-        return ['admin_ui_config' => $this->aggregator->getConfig()];
+        return [
+            'admin_ui_config' => $this->createConfigWrapper(),
+        ];
+    }
+
+    /**
+     * Create lazy loaded configuration.
+     *
+     * @return \EzSystems\EzPlatformAdminUi\UI\Config\ConfigWrapper
+     */
+    private function createConfigWrapper(): ConfigWrapper
+    {
+        $factory = new LazyLoadingValueHolderFactory();
+        $initializer = function (& $wrappedObject, LazyLoadingInterface $proxy, $method, array $parameters, & $initializer) {
+            $initializer = null;
+            $wrappedObject = new ConfigWrapper($this->aggregator->getConfig());
+            return true;
+        };
```

Comment:
Code looks kinda scary ;) 
but thanks for fixing this @adamwojs, I do a lot of "no-database" composer installs :)